website/content/en/highlights/2022-11-07-0-26-0-upgrade-guide.md

```diff
@@ -66,3 +67,11 @@ get the same functionality through a `remap` transform with [`geoip` enrichment
 tables](/docs/reference/configuration/global-options/#enrichment_tables.geoip). These can be used
 with [VRL's enrichment table functions](/docs/reference/vrl/functions/#enrichment-functions) to
 enrich events using a [GeoIP database](https://www.maxmind.com/en/geoip2-databases).
+
+#### Enrichment of `logstash` source with `source_type` field {#logstash-source-type-field}
+
+As part of the effort to add log namespacing support to all sources, in addition to setting the
+`source_type` to the Vector namespace, the Legacy namespace was also modified to enrich events
+with the global option [log_schema.source_type_key](https://vector.dev/docs/reference/configuration/global-options/#log_schema.source_type_key).
+
+This marks a breaking change to the shape of the log events output from the `logstash` source.
```

Comment:
I'd rewrite this to not actually mention the legacy vs vector namespace - we've not announced/documented/etc these yet so just saying something about including this field for consistency and all that jazz